sbin/start-history-server.sh

```diff
@@ -32,7 +32,7 @@ fi
 # Any changes need to be reflected there.
 CLASS="org.apache.spark.deploy.history.HistoryServer"
 
-if [[ "$@" = *--help ]] || [[ "$@" = *-h ]]; then
+if [[ "$*" = *--help ]] || [[ "$*" = *-h ]]; then
```

Comment:
SC2199 (error): Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

Reply:
Same with here. We actually want to check whether the args is end of `-h` or `--help`,